ante/ante.go

```diff
@@ -36,6 +37,9 @@ func NewAnteHandler(opts HandlerOptions) (sdk.AnteHandler, error) {
 	if opts.GlobalFeeSubspace.Name() == "" {
 		return nil, sdkerrors.Wrap(sdkerrors.ErrLogic, "param store is required for ante builder")
 	}
+	if opts.StakingSubspace.Name() == "" {
+		return nil, sdkerrors.Wrap(sdkerrors.ErrLogic, "param store is required for ante builder")
```

Comment:
```suggestion
		return nil, sdkerrors.Wrap(sdkerrors.ErrLogic, "staking param store is required for ante builder")
```

Reply:
also, instead return `sdkerrors.ErrLogic` we can create new custom errors for Gaia, like `gaiaerrors.ErrSubspaceNotFound`

Reply:
shall we use `sdk.ErrNotFound` instead?

Reply:
actually sdk uses `ErrLogic` in [ this situation](https://github.com/cosmos/cosmos-sdk/blob/18d8b64ce596131b986a96bfa44524cb9625970b/x/auth/ante/ante.go#L27).